### PR TITLE
Update veda-config issue template

### PR DESCRIPTION
**Pull request by aboydnw**
Updating the issue template in hopes that this repo can become the primary source of content requests and a way for us to track all the content requests using a new project board I have created.

## Why are you creating this Pull Request?

- [Adding Datasets, Discoveries or Thematic areas](?title=Content%3A%20%3Cname%3E&expand=1&template=content.md)
- [Version Release](?title=Deploy%20vX.X.X&expand=1&template=version_release.md)
- [Other](?expand=1&template=default.md)

### CODE REVIEW

**Comment by aboydnw** [comment]
@j08lue @danielfdsilva @faustoperez My idea for streamlining the content request flow is to start having people come directly to veda-config and create an issue. Then, I have created a [new project board](https://github.com/orgs/NASA-IMPACT/projects/68/views/1) to just track these content requests. I figure we can find time weekly to review the board, maybe 5 minutes during our weekly internal sync with IMPACT. My idea is that this would replace the[ discovery tracker,](https://docs.google.com/spreadsheets/d/1QF0BPooGhMFjvEsZwf-vxiirIGyVeYDVqDMvDfzBbfM/edit#gid=0) which hasn't really stuck, in large part because it's just not an artifact that gets brought up often and it's not connected to our workflow. I thought this would be closer to the actual configuration and was something we could bring up during our weekly syncs.

@j08lue I link to the almanac page for now in the template. When we have a more permanent location for that documentation, I'll update the template. 

@faustoperez this issue template and project board might be putting the cart before the horse, as I know it strongly relates to your research on the content configuration workflow. They are pretty easy to update, though, so I started putting them together and we can adjust later.

**Comment by j08lue** [comment]
Hmm, this looks to me more like docs than an issue template. Would contributors remove all this content when they open a ticket (hopefully after having read it)? And what would they keep that structures the newly opened ticket in a way we want?

For requests to ingest datasets into VEDA STAC, we currently also have a [template in veda-data-pipelines](https://github.com/NASA-IMPACT/veda-data-pipelines/blob/ac465e17e0bc2615754980482895f3ddedf21f2a/.github/ISSUE_TEMPLATE/add-new-dataset-simple.md) and track dataset ingestion jobs [as issues in that repo](https://github.com/NASA-IMPACT/veda-data-pipelines/issues?q=is%3Aissue+is%3Aopen+label%3Adataset). 

There is also an effort going on to separate dataset/collection definitions from the code for `veda-data-pipelines`: https://github.com/NASA-IMPACT/veda-architecture/issues/246, which will probably include moving the ingestion requests from that repo to somewhere else. Perhaps valuable to coordinate with @ividito, so we find a good solution that combines "Dashboard content" and "STAC content".

**Comment by aboydnw** [comment]
That is a good point @j08lue I had not included much in the way of instructions. My idea is that this could serve as a portal to documentation and also a reference for how to get started. Then, it could be an overarching issue to track throughout the entire process that we can reference.

But I don't want to put too much more effort into this until we are able to sync with @faustoperez @jsignell and @ividito 
If there is a better solution that doesn't involve Github, I'm all for it. But I am hoping to have some way to track all of these threads in one place, which is why I started here.

File: .github/ISSUE_TEMPLATE/configure-xx-story-for-the-dashboard.md [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
+
+1. New data that does not yet exist in VEDA
+2. New Discoveries that only involve graphics or maps of [data already in VEDA](http://veda-staging-stac-browser.s3-website-us-west-2.amazonaws.com/)
+3. New Discoveries that involve datasets that don't yet exist in VEDA
+
+If your content falls into option 1, fill in Dataset in the Title of this ticket. If it is option 2, fill in Discovery. If option 3, fill in Both. This means you will have to ingest the data before completing the configuration of the discovery.
+
+### Configuration Process
+
+Follow the high-level [instructions](https://almanac.io/handbook/veda-product-development-documentation-I3UwKo/getting-from-ingestion-to-discovery-I1ESDqS4NuHVdOx82peffQqMGDDdTyE4) linked above to ingest data or configure the dashboard. You should have the ability to create pull requests in all the necessary Github repositories. To create a pull request, it is recommended to do so from this issue, so that your pull requests will all be linked with this issue. This makes it easy to track for yourself and the VEDA team. To create a pull request from an issue, follow these instructions.
```
Comment (hanbyul-here) [inline .github/ISSUE_TEMPLATE/configure-xx-story-for-the-dashboard.md:26]: This instruction moved to : https://nasa-impact.github.io/veda-docs/contributing/dashboard-configuration/

**Comment by j08lue** [comment]
I think this is superseded by docs getting moved to `veda-docs` and cleanup of docs in `veda-ui` and `veda-config` that happened a while back. Closing.

**Pull request closed without merging**